src/main/java/org/jenkinsci/plugins/ParameterizedRemoteTrigger/utils/HttpHelper.java

```diff
@@ -592,46 +583,34 @@ private static ConnectionResponse tryCall(String urlString, String method, Build
 		}
 	}
 
+	private static Auth2 effectiveAuth(BuildContext context, Auth2 overrideAuth) {
+		if (overrideAuth != null && !(overrideAuth instanceof NullAuth)) {
+			// use Authorization Header if configured locally
```

Comment:
BTW I noticed that this class is using TABs rather than spaces, which was confusing and required extra effort to match. Recommend https://github.com/jenkinsci/plugin-pom/blob/master/README.md#formatting